docs/methoddocs/types.md

```diff
@@ -4,7 +4,7 @@
 
 ```{eval-rst}
 .. automodule:: ape.types
-    :members: BlockID, AddressType, ContractLog
+    :members: BlockID, AddressType, ContractLog, MockContractLog
```

Comment:
Do we want to add `BaseContractLog` here as well?

Reply:
Added